ParquetClassLibrary/EntityID.cs

```diff
@@ -1,4 +1,5 @@
 using System;
```

Comment:
nit: Arbitrarily picking this file to represent this; can the using statements be alphabetized?

Reply:
Mm, that's probably a good idea.  I've been using ReSharper's default organization scheme.  I will look into this later on down the line.